examples/emodel_pipeline_local_python/create_venv.sh

```diff
@@ -0,0 +1,9 @@
+deactivate
+module purge all # comment out this line if the script is not running on BB5
```

Comment:
maybe you can say 'if you are not using spack packages' instead of 'running on BB5'